phpstan-baseline.neon

```diff
@@ -0,0 +1,27 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Parameter \\#1 \\$str of function sanitize_text_field expects string, array\\<string\\>\\|string given\\.$#"
+			count: 1
+			path: src/Admin.php
+
+		-
+			message: "#^Constant WP_CONTENT_DIR not found\\.$#"
```

Comment:
We'll address this on separate PR - this will also be flagged by WP VIP coding standards